views/update/beneficial-owner-bo-mo-review.html

```diff
@@ -0,0 +1,37 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  {% include "includes/page-title.html" %}
+{% endblock %}
+
+{% block backLink %}
+  {% include "includes/back-link.html" %}
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div class="bo-mo-review-card">
+      <span class="govuk-caption-xl govuk-!-padding-bottom-1">{{ appData.entity_name }} - {{ appData.entity_number }}</span>
+      <h1 class="govuk-heading-xl">You're about to review beneficial owner and managing officer information</h1>
+
+      <p class="govuk-body">We'll show you the information we currently hold about the overseas entity's beneficial owners and managing officers.</p>
+      
+      <p class="govuk-body">You'll need to:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>check the information is correct, and update it if needed</li>
+        <li>enter the home address for individual beneficial owners and managing officers</li>
+        <li>tell us if they are no longer a beneficial owner or managing officer, if relevant</li>
+        <li>add any new beneficial owners or managing officers, if relevant</li>
+      </ul>
+
+      <br/>
+
+      <a href="{{OE_CONFIGS.UPDATE_BENEFICIAL_OWNER_TYPE_URL}}"  id="continue" role="button" draggable="false" class="govuk-button" data-module="govuk-button">
```

Comment:
There is a "continue-button" html file that already exists, can we reuse it here?

Reply:
I agree with this